### PR TITLE
PTM-99 Ignore unknown patients when finding matches

### PR DESCRIPTION
**Pull request by wluyima**
https://issues.openmrs.org/browse/PTM-99

### CODE REVIEW

File: omod/src/main/resources/config.xml [hunk 1]
```diff
@@ -59,6 +59,13 @@
         </description>
     </globalProperty>
 
+    <globalProperty>
+        <property>patientmatching.unknownPatientAttributeTypeUuid</property>
+        <description>The uuid of the person attribute type that is used to mark a patient as unknown, this assumes the
```
Comment (dkayiwa) [inline omod/src/main/resources/config.xml:64]: What does `where true is the value for the unknown patient` mean?

Comment (wluyima) [reply omod/src/main/resources/config.xml:64]: Meaning if the person attribute value is set true

Comment (dkayiwa) [reply omod/src/main/resources/config.xml:64]: Do you mind making it clearer?

Comment (wluyima) [reply omod/src/main/resources/config.xml:64]: Done

**Pull request merged**